src/GafferArnoldUI/GoboVisualiser.cpp

```diff
@@ -36,9 +36,11 @@
 
 #include "GafferOSL/ShadingEngine.h"
 
-#include "GafferSceneUI/LightFilterVisualiser.h"
+#include "GafferSceneUI/Export.h"
```

Comment:
And this one.

Reply:
https://github.com/GafferHQ/gaffer/commit/e4be79c410ad38b91127b9d2120aab37fa134c06